content/intro-to-storybook/ember/en/get-started.md

```diff
@@ -0,0 +1,101 @@
+---
+title: 'Storybook for Ember tutorial'
+tocTitle: 'Get started'
+description: 'Setup Storybook in your development environment'
+---
+
+Storybook runs alongside your app in development mode. It helps you build UI components isolated from the business logic and context of your app. This edition of Learn Storybook is for Ember; other editions exist for [React](/react/en/get-started), [React Native](/react-native/en/get-started), [Vue](/vue/en/get-started), [Angular](/angular/en/get-started) and [Svelte](/svelte/en/get-started).
+
+![Storybook and your app](/intro-to-storybook/storybook-relationship.jpg)
+
+## Setup Ember Storybook
+
+We’ll need to follow a few steps to get the build process set up in your environment. To start with, we want to use [ember-cli](https://github.com/ember-cli/ember-cli) to setup our build system, and enable [Storybook](https://storybook.js.org/) and [Qunit](https://qunitjs.com/) testing in our app. Let’s run the following commands:
+
+```bash
+# Create our application:
+ember new taskbox
+
+cd taskbox
+
+# Add Storybook:
+npx -p @storybook/cli sb init
```

Comment:
Should

Reply:
@tmeasday sorry about the should. That sentence needs a little more attention, when i was going over what might be the issue with the ember generate problem and why wasn't Storybook being bootstrapped after the addon was installed  (which would be a nice feature to add to cut down the number of commands and further streamline the user experience) i left it there. I'm going to update the wording shortly

Reply:
@tmeasday this should be fixed with the latest commit.

Reply:
Hey @jonniebigodes I have no idea what this comment meant, sorry I think it was a typo!

Reply:
I think this line is executed with the next one (if the bug is fixed). `ember install @storybook/ember-cli-storybook` runs the storybook initializer? Doesn't it? I never used that line to setup storybook for any of my ember projects. Just call the ember generator.

PS. Yes, there is a bug with it atm.

Reply:
@gossi hence why i added both commands. I started out just adding the addon and i thought i was home free...when i tried to initialize it, zoinks....some errors popped in. @tmeasday no worries.